README.md

```
@@ -1,151 +1,210 @@
+cargo build
+
+# Release build
+cargo build --release
+
+# Run tests
+cargo test
+```
+
+## License
+
+MIT
```

markdownlint-cli2 (0.20.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)